nutanix/resource_nutanix_network_security_rule.go

```diff
@@ -44,6 +43,12 @@ func resourceNutanixNetworkSecurityRule() *schema.Resource {
 				Type:     schema.TypeString,
 				Computed: true,
 			},
+			"wait_timeout": {
+				Type:         schema.TypeInt,
+				Optional:     true,
+				Default:      DEFAULTWAITTIMEOUT,
+				ValidateFunc: validation.IntAtLeast(MINIMUMWAITTIMEOUT),
+			},
```

Comment:
Should this be changed to `Timeouts` attribute?

Reply:
Yeah. I missed that. Now it's done.